### ISSUE TITLE
Universal Link not opening app in ios

### ISSUE THREAD

**Issue by Andranjo97** [RodrigoSMarques/flutter_branch_sdk#44]
Hello @RodrigoSMarques,

I have set up the iOS part of branch. It is working fine and I can generate shareable links. However, when I try to open one of the generated links, the user does not get redirected to the app but to Safari. I have deployed to TestFlight just in case my app was not recognized as installed but did not work either. Do you know what could be happening?
Flutter version: 1.20.1
Flutter Branch sdk: 1.0.0

**Comment by RodrigoSMarques** [comment]
Hi @Andranjo97 
Did you complete all the configuration steps on the Dashboard?
Did you fill in all the information for your IOS project?

See the instructions for these two links:
https://help.branch.io/developers-hub/docs/ios-testing
https://help.branch.io/developers-hub/docs/ios-troubleshooting

**Comment by Andranjo97** [comment]
@RodrigoSMarques yep I completed all the configurations on the Dashboard. I just double-checked. And, I did fill the info for my iOS project (branch links, entitlements, etc). Thanks for the sources! After using `My First Link` from my Dashboard as mentioned [here](https://help.branch.io/developers-hub/docs/ios-testing#section-test-deep-link). But, it worked for the first time, and it did not work anymore. BTW, I'm not getting the message I should from this [section](https://help.branch.io/developers-hub/docs/ios-troubleshooting#section-validate-if-aasa-file-successfully-downloaded), even after restarting the device as suggested there.
